From: Petter Reinholdtsen <pere@hungry.com>
Date: Mon, 23 May 2022 08:49:20 -0600
Subject: Adjust menu location of program
Forwarded: not-needed
Reviewed-By: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2024-04-28

Add more categories to the desktop file to adjust where
Cura show up in the GUI menu.
---
 com.ultimaker.cura.desktop.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/com.ultimaker.cura.desktop.in b/com.ultimaker.cura.desktop.in
index 1e17e30..bdd91c0 100644
--- a/com.ultimaker.cura.desktop.in
+++ b/com.ultimaker.cura.desktop.in
@@ -14,6 +14,6 @@ Icon=cura-icon
 Terminal=false
 Type=Application
 MimeType=model/stl;application/vnd.ms-3mfdocument;application/prs.wavefront-obj;image/bmp;image/gif;image/jpeg;image/png;text/x-gcode;application/x-amf;application/x-ply;application/x-ctm;model/vnd.collada+xml;model/gltf-binary;model/gltf+json;model/vnd.collada+xml+zip;
-Categories=Graphics;
+Categories=Graphics;Education;Development;Science;
 Keywords=3D;Printing;Slicer;
 StartupWMClass=cura.real
